project/MultiScalaProject.scala

```diff
@@ -0,0 +1,96 @@
+package build
+
+import sbt._
+import Keys._
+
+final class MultiScalaProject private (private val projects: Map[String, Project]) {
```

Comment:
`MultiScalaProject` could extend `sbt.CompositeProject` and implement the following method:
```scala
override def componentProjects: Seq[Project] = projects.valuesIterator.toSeq
```
so that you don't need to triplicate all the `lazy val`s in the `build.sbt`. This is what `CrossProject` does now:
https://github.com/portable-scala/sbt-crossproject/blob/master/sbt-crossproject/src/main/scala/sbtcrossproject/CrossProject.scala#L17